Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.Styles/App_LocalResources/Styles.resx

```diff
@@ -0,0 +1,273 @@
+    <xsd:element name="root" msdata:IsDataSet="true">
+      <xsd:complexType>
+        <xsd:choice maxOccurs="unbounded">
+          <xsd:element name="metadata">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" />
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="nav_Styles.Text" xml:space="preserve">
+    <value>Styles</value>
+  </data>
+  <data name="ModuleDescription" xml:space="preserve">
+    <value>DNN Provides css-varialbes (custom properties) to designers and developers to allow them to style their solutions to the site branding preferences. These style preferences may or may not be implemented by a theme or extension.</value>

```

Comment:
```suggestion
    <value>DNN provides css-variables (custom properties) to designers and developers to allow them to style their solutions to the site branding preferences. These style preferences may or may not be supported by a theme or extension.</value>
```

There is a typo in variables, but I'm wondering if there is a better way to emphasize that these colors will not impact anything if they are not adjusted.

Is there any impact to DNN regardless of theme usage?  If so, we should call that out here as well

Reply:
Well yes, the goal is to harmonize the UI and provide a single source of truth for such preferences. So it will affect all of DNN UI components, persona bar, etc. the new default CSS will use them. It would make no sense to have an option to not inject them. It will not affect 3rd party solutions that don't use them or the older UI/UX patterns. The defaults should be close the the existing Dnn branding so unless they are changed, it won't look too far from what it looks now. The exception to that is we currently have different styles in the persona bar than it the rest of the editing UI. With the new default.css, the "others" will be more aligned with the current person's bar styling (flatter).

Reply:
But if a theme does unload default.css and does its own thing, that wins... So that would be the "opt-out" exit route as in the past.